hack/update-attributions.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -eoux pipefail
+
+SPLIT="***"
+
+cat <<EOF > ATTRIBUTIONS.md
+# Attributions
+
+This application uses Open Source components. You can find the source
+code of their open source projects along with license information below.
+We acknowledge and are grateful to these developers for their contributions
+to open source.
+
+## libssh2
+
+Libssh2 was obtained in source-code form from its github repository: 
+https://github.com/libssh2/libssh2/
+
+No changes were made to its original source code. 
+
+Copyright notice (https://raw.githubusercontent.com/libssh2/libssh2/main/COPYING):
```

Comment:
```suggestion
Copyright notice (https://raw.githubusercontent.com/libssh2/libssh2/master/COPYING):
```
Invalid link. Must be a result of find and replace :)

Reply:
Nice spot, thank you - just amended it.

Reply:
@darkowlzz PTAL